src/Installers/Windows/WindowsHostingBundle/Bundle.wxs

```diff
@@ -57,13 +57,126 @@
         <!-- Bundle variables -->
         <Variable Name="BundleNameShort" Value="$(var.BundleNameShort)"/>
         <Variable Name="BundleNameSub" Value="$(var.BundleNameSub)"/>
-        <Variable Name="OPT_NO_ANCM" Value="0" bal:Overridable="yes"/>
-        <Variable Name="OPT_NO_FTS" Value="0" bal:Overridable="yes"/>
-        <Variable Name="OPT_NO_LTS" Value="0" bal:Overridable="yes"/>
-        <Variable Name="OPT_NO_SHAREDFX" Value="0" bal:Overridable="yes"/>
-        <Variable Name="OPT_NO_RUNTIME" Value="0" bal:Overridable="yes"/>
-        <Variable Name="OPT_NO_X86" Value="0" bal:Overridable="yes"/>
-        <Variable Name="OPT_NO_SHARED_CONFIG_CHECK" Value="0" bal:Overridable="yes" />
+        <Variable Name="OPT_NO_ANCM" bal:Overridable="yes"/>
+        <Variable Name="OPT_NO_FTS" bal:Overridable="yes"/>
+        <Variable Name="OPT_NO_LTS" bal:Overridable="yes"/>
+        <Variable Name="OPT_NO_SHAREDFX" bal:Overridable="yes"/>
+        <Variable Name="OPT_NO_RUNTIME" bal:Overridable="yes"/>
+        <Variable Name="OPT_NO_X86" bal:Overridable="yes"/>
+        <Variable Name="OPT_NO_SHARED_CONFIG_CHECK" bal:Overridable="yes" />
+
+        <!-- Use values from the registry for switches, if the registry values are set & the user hasn't passed any switches from the command line -->
+
+        <util:RegistrySearch Id="opt_no_ancm_should_be_set" 
+                         Condition="NOT OPT_NO_ANCM AND NOT OPT_NO_ANCM=0 AND NOT OPT_NO_FTS AND NOT OPT_NO_FTS=0 AND NOT OPT_NO_LTS AND NOT OPT_NO_LTS=0 AND NOT OPT_NO_SHAREDFX AND NOT OPT_NO_SHAREDFX=0 AND NOT OPT_NO_RUNTIME AND NOT OPT_NO_RUNTIME=0 AND NOT OPT_NO_X86 AND NOT OPT_NO_X86=0 AND NOT OPT_NO_SHARED_CONFIG_CHECK AND NOT OPT_NO_SHARED_CONFIG_CHECK=0"
```

Comment:
Find a way to put this condition in one place

Reply:
See comment below